src/core/ext/transport/chttp2/transport/flow_control.cc

```diff
@@ -175,7 +175,10 @@ int64_t TransportFlowControl::target_window() const {
 }
 
 FlowControlAction TransportFlowControl::UpdateAction(FlowControlAction action) {
-  if (announced_window_ < target_window() / 2) {
```

Comment:
how about `announced_window_ < (target_window() + 1) / 2` ?
This seems more standard to me